assets/terraform-modules/aws/flatcar-linux/kubernetes/variables.tf

```
@@ -196,3 +196,8 @@ variable "ignore_x509_cn_check" {
   type        = bool
   default     = false
 }
+
+variable "conntrack_max_per_core" {
+  description = "--conntrack-max-per-core value for kube-proxy. Maximum number of NAT connections to track per CPU core (0 to leave the limit as-is and ignore conntrack-min)."
```

I don't think the `ignore conntrack-min` makes sense in this context, or at least I don't understand what it means. Can we explain this better or remove it? Maybe say that this will ignore the conntrack-min option in kube-proxy that has a default of X?

---

@surajssd requested copying Kubernetes docs here: https://github.com/kinvolk/lokomotive/pull/1187#discussion_r526853955

---

I think we should at least mention this is a kube-proxy flag. How about this?

```suggestion
  description = "--conntrack-max-per-core value for kube-proxy. Maximum number of NAT connections to track per CPU core (0 to leave the limit as-is and ignore the conntrack-min kube-proxy flag)."
```

---

I already dislike the idea of copying the docs. Modifying them on top is even worse :see_no_evil: @surajssd ?

---

How about just saying this? It's not copying neither modifying. It's creative work :P

```suggestion
  description = "--conntrack-max-per-core value for kube-proxy. It controls the maximum number of NAT connections to track per CPU core, see the kube-proxy documentation for details."
```

---

Will apply what suggested.